<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CsSharkü¶à</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTableWidget" name="tableWidget"/>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">Ëß£ÊûêÂ¶Ç‰∏ãÔºö</string>
           </property>
          </column>
         </widget>
         <widget class="QTextBrowser" name="textBrowser"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuproject">
    <property name="title">
     <string>È°πÁõÆ</string>
    </property>
   </widget>
   <widget class="QMenu" name="menurun">
    <property name="title">
     <string>ËøêË°å</string>
    </property>
    <addaction name="actionrunandstop"/>
    <addaction name="actionclear"/>
   </widget>
   <addaction name="menuproject"/>
   <addaction name="menurun"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionrunandstop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/start.png</normaloff>:/Image/start.png</iconset>
   </property>
   <property name="text">
    <string>ËøêË°å</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/empty.png</normaloff>:/Image/empty.png</iconset>
   </property>
   <property name="text">
    <string>Ê∏ÖÁ©∫</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
